praatio/tgio.py

```diff
@@ -1883,7 +1883,7 @@ def _parseNormalTextgrid(data):
     newTG = Textgrid()
 
     # Toss textgrid header
-    header, data = data.split("item [", 1)
+    header, data = re.split(r'item ?\[', data, maxsplit=1, flags=re.MULTILINE)
```

Comment:
From what I read, MULTILINE mode only affects the interpretation of `^` and `$` in a regex.  If thats the case, then this MULTILINE can be removed I think?